docs/building-deployer.md

```diff
@@ -43,6 +43,26 @@ on your application.
 For information on using `mpdev` to test your application, read the
 [`mpdev` reference](mpdev-references.md).
 
+### Setting smoke test timeouts
```

Comment:
Remove "smoke test" since WAIT_FOR_READY_TIMEOUT is not test related.

Reply:
Removed smoke tests from section title